include/libdnf5/transaction/transaction_history.hpp

```diff
@@ -73,10 +73,21 @@ class TransactionHistory {
     /// @since 5.0
     libdnf5::BaseWeakPtr get_base() const;
 
+    /// Get reason for package specified by name and arch at a point in history
+    /// specified by transaction id.
+    ///
+    /// @param name Name of rpm package
+    /// @param arch Arch of rpm package
+    /// @return Reason of the last transaction item before transaction_id_point that
```

Comment:
Parameter `transaction_id_point` is not described. May I ask you to add a description with a hint how to obtain it?

Reply:
Thank you, added.